packages/inventory-compliance/src/RulesComplianceFilter.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { FilterDropdown } from '@redhat-cloud-services/frontend-components';
+
+const FILTER_CATEGORIES = [
+    {
+        type: 'checkbox', title: 'Passed', urlParam: 'passed', values: [
```

Comment:
Updated mock for this. Uses radios and a little more description
![image](https://user-images.githubusercontent.com/23172038/58733614-fcd19b00-83c2-11e9-8855-b41552097a3d.png)

Reply:
Fixed